doc/developer_documentation.md

```diff
@@ -532,6 +537,26 @@ pip install -e .
 # stim is now installed in current virtualenv as dev reference
 ```
 
+## <a name="pypackage.stim.cmake"></a>python packaging `stim` with cmake
+
+A python module can be built using cmake. The output can be imported as `import stim`, but is not packaged, so if you want to pip install, prefer building the bindings with Bazel.
+
+```bash
+# from the repository root
+cmake stim_python_bindings
+# output is at out/stim.cpython-${PYTHON_VERSION}-${ARCH}.so
+```
+to import the module we need to add `out` to the python import path:
+```python
+# from the repository root
+# python
+import sys
+import os
+sys.path.append(os.getcwd() + '/out')
```

Comment:
I would explain this as "run `PYTHON_PATH=out python -c "import stim; print(stim__version__)"`" instead of messing with python's internal sys stuff

Reply:
ah nice. I wasn't aware that you could set is using an environment variable. Updated